building/tooling/analyzers/feedback-guidelines.md

```diff
@@ -4,8 +4,8 @@ _NOTE: This spec is currently being updated._
 
 The goal of a language track on Exercism is to give people a way to achieve a
 high level of [fluency](/docs/using/product/fluency) at a low level of proficiency. We're aiming for fluency
-in the syntax, idioms, and the standard library of the language. You can read
-more about the [goal of exercism here](https://github.com/exercism/docs/blob/master/about/goal-of-exercism.md).
+in the syntax, idioms, and the standard library of the language. Read
+more about the [goal of Exercism here](https://github.com/exercism/docs/blob/master/about/goal-of-exercism.md).
```

Comment:
This link currently doesn't lead anywhere, see #424. Linking to *getting started* could be a potential fix, or revert these current changes so as to not cause a merge conflict and let the latter PR resolve it.

```suggestion
in the syntax, idioms, and the standard library of the language. Read
more about the [goal of Exercism here](/docs/using/getting-started).
```

Reply:
Are you sure the getting started page is a valid link for "goal of Exercism"? Also, which changes should I revert to ensure that there isn't a merge conflict?

Reply:
It should probably link here: `https://exercism.org/about/`